t/008-log.t

```diff
@@ -938,3 +938,27 @@ GET /test
 ]
 --- no_log eval
 "you can't see me init_worker"
+
+
+=== TEST 25: debug log level for nginx error.log
```

Comment:
What is the default log level in test::nginx?

Reply:
The default log level is crit, I redefined the log level of error.log to crit in this location.